src/modules/api.js

```diff
@@ -0,0 +1,54 @@
+const form = document.querySelector('.form');
+const refreshBtn = document.getElementById('refresh-btn');
+const table = document.getElementById('recent-list');
+const inputName = document.getElementById('name');
+const inputScore = document.getElementById('score');
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/iNLPApgiyuuDUgI7yzuD/scores/';
+
+const uploadScore = async (user, score) => {
+  const response = await fetch(`${url}`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ user, score }),
+  });
+  const data = await response.json();
+  return data;
+};
+
+const getScores = async () => {
+  const response = await fetch(`${url}`);
```

Comment:
- [ ] **[OPTIONAL]** Since you're not adding extra info to the URL string, it is safe to remove the template string, something like this: 🤖 🔧 

```suggestion
  const response = await fetch(url);
```